iosxr_store_box.py

```diff
@@ -37,11 +37,24 @@
 import argparse
 from argparse import RawDescriptionHelpFormatter
 import logging
+import hashlib
```

Comment:
@glennmatthews Sorry Glenn didn't even write a US for this. Was an urgent ask from Jose who's talking to infosec.